llama_cpp/llama_cpp.py

```diff
@@ -25,6 +25,12 @@ def _load_shared_library(lib_base_name):
         _base_path / f"{lib_base_name}{lib_ext}"
     ]
 
+    if ("LLAMA_LIB" in os.environ):
```

Comment:
@SagsMug great work, can we change the environment variable to `LLAMA_CPP_LIB`

Reply:
> @SagsMug great work, can we change the environment variable to `LLAMA_CPP_LIB`

Done!